applications/openshift/kubernetes/master/scheduler/address_argument/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Ensure that the --address argument is set to 127.0.0.1'
```

Comment:
Change this to `Ensure that the --address argument is set` or `Ensure That The --address Argument Is Configured`